test/parallel/test-stream-duplex-from.js

```diff
@@ -142,7 +142,9 @@ const { Blob } = require('buffer');
       }
       assert.strictEqual(ret, 'abcdefghi');
     },
-    common.mustCall(),
+    common.mustCall((error) => {
+      assert.ok(error === undefined);
+    }),
```

Comment:
```suggestion
    common.mustSucceed(),
```

Reply:
Didn't know about that method. Ty for the suggestion.